src/org/labkey/test/tests/issues/IssueAPITest.java

```diff
@@ -0,0 +1,377 @@
+package org.labkey.test.tests.issues;
+
+import org.junit.BeforeClass;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.issues.GetIssueCommand;
+import org.labkey.remoteapi.issues.GetIssueResponse;
+import org.labkey.remoteapi.issues.IssuesCommand;
+import org.labkey.remoteapi.issues.IssueModel;
+import org.labkey.remoteapi.issues.IssueResponse;
+import org.labkey.remoteapi.issues.IssueResponseModel;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.TestFileUtils;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.InDevelopment;
+import org.labkey.test.util.APIUserHelper;
+import org.labkey.test.util.ApiPermissionsHelper;
+import org.labkey.test.util.IssuesHelper;
+import org.labkey.test.util.PermissionsHelper;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.labkey.test.util.PasswordUtil.getUsername;
+
+@Category({InDevelopment.class})
+public class IssueAPITest extends BaseWebDriverTest
+{
+    IssuesHelper _issuesHelper = new IssuesHelper(this);
+    APIUserHelper _userHelper = new APIUserHelper(this);
+    static String ISSUES = "issues";
+    static Long TEST_USER_ID;
+    static String TEST_USER_NAME;
+    static String TEST_USER_DISPLAY_NAME;
+
+    static Long TEST_BUDDY_ID;
+    static String TEST_BUDDY_NAME = "testbuddy@issues.test";
+    static String TEST_BUDDY_DISPLAY_NAME;
+
+    @Override
+    protected void doCleanup(boolean afterTest) throws TestTimeoutException
+    {
+        super.doCleanup(afterTest);
+    }
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        IssueAPITest init = (IssueAPITest) getCurrentTest();
+        init.doSetup();
+    }
+
+    private void doSetup()
+    {
+        _containerHelper.createProject(getProjectName(), null);
+        _issuesHelper.createNewIssuesList(ISSUES, getContainerHelper());
+
+        TEST_USER_NAME = getUsername();
+        TEST_USER_ID = Long.valueOf(_userHelper.getUserId(TEST_USER_NAME));
+        TEST_USER_DISPLAY_NAME = _userHelper.getDisplayNameForEmail(TEST_USER_NAME);
+        _userHelper.createUser(TEST_BUDDY_NAME);
+        TEST_BUDDY_ID = Long.valueOf(_userHelper.getUserId(TEST_BUDDY_NAME));
```

Comment:
`createUser` returns an object containing the user ID.
```suggestion
        TEST_BUDDY_ID = _userHelper.createUser(TEST_BUDDY_NAME).getUserId().longValue();
```

Reply:
very cool!  I like that a lot